sys/fcntl.h is not a proper header file on musl:
```
/usr/include/sys/fcntl.h:1:2: warning: #warning redirecting incorrect #include <sys/fcntl.h> to <fcntl.h> [-Wcpp]
    1 | #warning redirecting incorrect #include <sys/fcntl.h> to <fcntl.h>
      |  ^~~~~~~
```
not an issue, just gets rid of an easy warning

diff --git a/tests/utils.h b/tests/utils.h
index ee2ec03..0bd7bfa 100644
--- a/tests/utils.h
+++ b/tests/utils.h
@@ -31,3 +31,2 @@
 #include <fcntl.h>
-#include <sys/fcntl.h>
 #include <sys/stat.h>
